great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -375,7 +374,7 @@ def _get_count_query_mssql(
         ignore_values_condition: BinaryExpression,
     ) -> Select:
         # mssql expects all temporary table names to have a prefix '#'
-        temp_table_name: str = f"#ge_tmp_{str(uuid.uuid4())[:8]}"
+        temp_table_name: str = generate_temporary_table_name()
```

Comment:
I believe this should be `generate_temporary_table_name(default_table_name_prefix="#ge_temp_")`. I wonder why this is passing tests?

Reply:
@NathanFarmer Not sure, but I think that you are correct -- let me fix it.  Great catch.  Thank you!

Reply:
Done -- please check.  Thanks.